icu4c/source/test/intltest/numbertest_decimalquantity.cpp

```diff
@@ -467,4 +469,191 @@ void DecimalQuantityTest::testNickelRounding() {
     status.expectErrorAndReset(U_FORMAT_INEXACT_ERROR);
 }
 
+void DecimalQuantityTest::testCompactDecimalSuppressedExponent() {
+    IcuTestErrorCode status(*this, "testCompactDecimalSuppressedExponent");
+    Locale ulocale((const char *) "fr-FR");
```

Comment:
I don't think you need the cast on this line

Reply:
Done.